userguide/services.rst

```diff
@@ -1537,15 +1538,6 @@ a multiple of _SC_PAGESIZE (typically *4096*) to avoid memory
 fragmentation. This will increase Samba's memory requirements and
 should not be used on systems with limited RAM.
 
-To increase network performance, read the Samba section on socket
-options in the
-`smb.conf manual page
-<https://www.freebsd.org/cgi/man.cgi?query=smb.conf&manpath=FreeBSD+11.0-RELEASE+and+Ports>`_.
-It indicates which options are available and recommends that you
-experiment to see which are supported by your clients and improve your
-network's performance.
-#endif freenas
-
```

Comment:
Heh. I'm kind of new to this git/github thing and didn't realize any additional edits I make get appended to existing pull requests. :-)

Adjusting socket options is somewhat frowned upon in the samba mailing lists (JRA has even mentioned ripping the parameter out of samba entirely). Most edits people do are voodoo, and some of them actually hurt performance. It's probably better to adjust send and receive buffers through sysctls (which people with Intel 10GB NICs tend to do). This is because it will still allow the kernel to automatically tune send and receive buffers. 

Manipulating IPTOS headers may help some people who have samba passing through routers (and have configured QOS in routers accordingly), but I believe this probably is a more advanced topic than most people looking at the FreeNAS handbook are trying to address.

TL;DR, socket options aren't exactly where I'd point people to if they were trying to improve samba performance.

I'll fix the #endif freenas. I'm still learning the ropes of the markup language. I'll also leave the blurb about socket options if you want ;-)